language-support/ts/daml-react/index.test.ts

```diff
@@ -351,6 +387,9 @@ describe('useStreamQuery', () => {
     mockStreamQuery.mockClear();
     mockStreamQuery.mockReturnValueOnce(stream);
     act(() => result.current.setQuery(query2));
+    // live event
+    act(() => void emitter.emit('live', []));
```

Comment:
`[]` is not a valid state for a streaming fetch-by-key.
```suggestion
    act(() => void emitter.emit('live', null));
```